update cfp with new closing date

ei8fdb opened the pull request:
extending cfp closing date.

ei8fdb committed:
update cfp with new closing date

evalica commented on _posts/2019-02-02-opensourcedesign-fosdem.md:
```diff
@@ -52,7 +52,7 @@ By submitting a proposal you consent to be recorded and agree to license the con
 #### Submission deadline:
 
-1st December 2018
+8th December 2018
```
You could have made it like: 
--1st December 2018-- (Extended: 8th December 2018)

ei8fdb replied:
good point. i'll update it now.

👍

ei8fdb replied:
used this GH syntax: https://webapps.stackexchange.com/questions/14986/strikethrough-with-github-markdown

ei8fdb committed:
Update 2019-02-02-opensourcedesign-fosdem.md

updating the closing date by adding strikethough for original date (<del>1st December 2018</del>)

jancborchardt reviewed (approved):
Seems good!
